src/components/elementView/elementTabContainer.tsx

```diff
@@ -173,7 +173,17 @@ class ElementTabContainer extends React.Component<
           }
         />
 
-        <Tab className="tab-header do-not-print" id="sites" title="Sites" />
+        <Tab
+          className="tab-header do-not-print"
+          id="sites"
+          title="Sites"
+          disabled={
+            !(
+              this.props.permissionState.admin ||
+              this.props.permissionState.asset_management
+            )
+          }
+        />
```

Comment:
needed to add disabling logic to Site tab header

Reply:
True dat